packages/framework/fluid-static/api-report/fluid-static.api.md

```diff
@@ -15,6 +15,12 @@ import { IFluidLoadable } from '@fluidframework/core-interfaces';
 import { IRuntimeFactory } from '@fluidframework/container-definitions';
 import { ISharedObjectKind } from '@fluidframework/shared-object-base';
 
+// @public (undocumented)
+export interface AzureContainerSchema extends ContainerSchema {
```

Comment:
We should document these. And for `minRuntimeVersion`, we should explicitly note what it means for a value to not be specified.